it/test/uk/gov/hmrc/soletraderidentificationfrontend/views/CaptureOverseasTaxIdentifierViewTests.scala

```diff
@@ -67,16 +66,27 @@ trait CaptureOverseasTaxIdentifierViewTests {
       doc.getH1Elements.text mustBe messages.title
     }
 
-    "have the correct hint text" in {
-      val hints: Elements = doc.getHints
+    "have the correct paragraph text" in {
```

Comment:
I'd be inclined to get a list of the paragraph texts and check for the specific paragraph.